api/src/services/report/smsparser.js

```diff
@@ -209,9 +210,19 @@ const fieldParsers = {
   month: (raw) => {
     // keep months integers, not their list value.
     return parseNum(stripInvisibleCharacters(raw));
+  },
+  phone_number: (raw) => {
+    const formattedAndValidatedPhone = phoneNumberParser.normalize(config.getAll(), raw);
+    if (formattedAndValidatedPhone) {
+      return formattedAndValidatedPhone;
+    } else {
+      logger.error(`The provided phone number ${raw} is invalid`);
```

Comment:
Other validation failures are logged as `warn` - I think this is because it's not a failure of the code. I think we should keep this one consistent with those.

```suggestion
      logger.warn(`The provided phone number ${raw} is invalid`);
```